rmw_opensplice_cpp/src/rmw_node_names.cpp

```diff
@@ -25,15 +26,15 @@ extern "C"
 rmw_ret_t
 rmw_get_node_names(
   const rmw_node_t * /* node */,
-  rmw_string_array_t * /* node_names */)
+  utilities_string_array_t * /* node_names */)
 {
   RMW_SET_ERROR_MSG("get_node_names is not supported for Opensplice");
   return RMW_RET_ERROR;
 }
 
 rmw_ret_t
 rmw_destroy_node_names(
-  rmw_string_array_t * /* node_names */)
+  utilities_string_array_t * /* node_names */)
```

Comment:
I think this whole function should be removed now, see:

https://github.com/ros2/rmw_connext/pull/224/files#diff-becdf01440f1757df19f4775b534e066L1578